react/features/lobby/constants.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Hide these emails when trying to join a lobby
+ */
+export const HIDDEN_EMAILS = [ 'inbound-sip-jibri@jitsi.net', 'outbound-sip-jibri@jitsi.net' ];
```

Comment:
Don't we have a better way to filter this? Or will SIP-Jibri always set those as the participant email addresses?

Reply:
that's what I've been told

Reply:
Ack